src_Core/CPU/CPU_Decode_C.bsv

```diff
@@ -854,8 +854,29 @@ function Tuple2 #(Bool, Instr) fv_decode_C_ADDI16SP (MISA  misa,  Bit #(2)  xl,
    end
 endfunction
 
+// C.CIncOffsetImm16CSP: expands to CIncOffsetImm
+function Tuple2 #(Bool, Instr) fv_decode_C_CIncOffsetImm16CSP (MISA  misa,  Bit #(2)  xl, Bool cap_enc,  Instr_C  instr_C);
+   begin
+      // Instr fields: CI-type
+      match { .funct3, .imm_at_12, .rd_rs1, .imm_at_6_2, .op } = fv_ifields_CI_type (instr_C);
+      Bit #(10) nzimm10 = { imm_at_12, imm_at_6_2 [2:1], imm_at_6_2 [3], imm_at_6_2 [0], imm_at_6_2 [4], 4'b0 };
+
+      Bool is_legal = ((misa.c == 1'b1)
+                       && (op == opcode_C1)
+                       && (funct3 == funct3_C_ADDI16SP)
```

Comment:
C_CIncOffsetImm16CSP even though they're the same

Reply:
Oops, good catch